src/app/(admin)/new_dashboard/page.tsx

```diff
@@ -1,22 +1,25 @@
 "use client";
-import { useState } from "react";
 import PageContainer from "@/components/PageContainer";
 import DashboardCard from "./shared/Card";
 import MessageInput from "./components/MessageInput";
 import { Box } from "@mui/material";
 import ContactList from "./components/ContactList";
 import ChatHeader from "./components/ChatHeader";
 import MessageList from "./components/MessageList";
-import { Contact } from "./types";
 import { useChat, useContacts } from "./hooks";
+import { useSocket } from "@/hooks";
 
 const Page = () => {
-  const { contacts } = useContacts();
-  const { messages, newMessage, handleChange, handleSend } = useChat();
-  const [selectedContact, setSelectedContact] = useState<Contact | null>(
-    contacts[0]
-  );
-
+  const { contacts, selectedChat, handleChatSelect } = useContacts();
```

Comment:
There seems to be some sort of inconsistencies. With selected contacts and selected chats, you should stick to one of the variables so it'll be more consistent and clear what you're trying to do here.

For example in chat header you're using the selectedContact prop as selectedChat, so I think it'll be better to keep to selectedContact variables.